### PR TITLE
Change license header of SPI library

### PR DESCRIPTION
**Pull request by bigdinotech**
-change the license header of SPI library to the correct one which is
Apache 2.0

### CODE REVIEW

**Comment by bigdinotech** [comment]
@calvinatintel @eriknyquist @SidLeung please review

File: Quark_Sketch/src/arduino/libraries/SPI_LIB.h [hunk 1]
```diff
@@ -1,15 +1,17 @@
 /*
```
Comment (calvinatintel) [inline Quark_Sketch/src/arduino/libraries/SPI_LIB.h:1]: Just to confirm, you have re-written this file from scratch without referencing the old LGPL version right?

File: Quark_Sketch/src/arduino/libraries/SPI_LIB.cpp [hunk 2]
```diff
@@ -173,7 +170,6 @@ uint32_t SPIClass::transfer32(uint32_t data)
 	rbuff = (uint32_t*)rx_buffer;
 	return *rbuff;
 }
-// Preinstantiate Objects //////////////////////////////////////////////////////
```
Comment (calvinatintel) [inline Quark_Sketch/src/arduino/libraries/SPI_LIB.cpp:176]: Did you mean to include this change?

Comment (bigdinotech) [reply Quark_Sketch/src/arduino/libraries/SPI_LIB.cpp:176]: yes

File: Quark_Sketch/src/arduino/libraries/SPI_LIB.h (hunk 1 above)
Comment (bigdinotech) [reply Quark_Sketch/src/arduino/libraries/SPI_LIB.h:1]: @calvinatintel yes this was pretty much written from scratch only looking at the old GPL version to ensure that the API is compatible

Comment (bigdinotech) [reply Quark_Sketch/src/arduino/libraries/SPI_LIB.h:1]: it had the LGPL license because i thought we were going to go with LGPL

**Review by calvinatintel — APPROVED** [review]

**Comment by calvinatintel** [comment]
cherry-picked

**Pull request closed without merging**